### PR TITLE
Use “navigable”+“parent”, not “parent browsing context”; fix fetch() refs

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/w3c/webappsec-credential-management/issues/213

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webappsec-credential-management/pull/223.html" title="Last updated on Jul 12, 2023, 5:06 AM UTC (b0cb461)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webappsec-credential-management/223/c009d69...b0cb461.html" title="Last updated on Jul 12, 2023, 5:06 AM UTC (b0cb461)">Diff</a>

### CODE REVIEW

**Comment by sideshowbarker** [comment]
@domenic, can you please look this over when you can make time?

**Comment by sideshowbarker** [comment]
@annevk When you have time (particularly about the `fetch()` refs)

**Comment by ASISBusiness** [comment]
> Fixes https://github.com/w3c/webappsec-credential-management/issues/213
> 
> 
> 
> ***
> <a href="https://pr-preview.s3.amazonaws.com/w3c/webappsec-credential-management/pull/223.html" title="Last updated on Jul 7, 2023, 11:50 PM UTC (ddd5952)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webappsec-credential-management/223/c009d69...ddd5952.html" title="Last updated on Jul 7, 2023, 11:50 PM UTC (ddd5952)">Diff</a>

**Review by ASISBusiness — APPROVED** [review]

File: index.bs [hunk 1]
```diff
@@ -76,6 +76,7 @@ spec: web-otp; urlPrefix: https://wicg.github.io/web-otp
 <pre class="link-defaults">
 spec:html; type:dfn; for:html-origin-def; text:origin
 spec:html; type:dfn; for:environment settings object; text:global object
+spec:html; type:dfn; for:navigable; text:parent
```
Comment (domenic) [inline index.bs:79]: If you use `[=navigable/parent=]` instead of `[=parent=]`, you can avoid this line.

File: index.bs [hunk 2]
```diff
@@ -294,13 +295,13 @@ spec:css-syntax-3;
 
     4.  Let |origin| be |settings|' [=environment settings object/origin=].
 
-    5.  Let |current| be |document|'s [=document/browsing context=].
+    5.  Let |navigable| be |document|'s [=node navigable=].
 
-    6.  While |current| has a [=parent browsing context=]:
+    6.  While |navigable| has a [=parent=]:
```
Comment (domenic) [inline index.bs:300]: Technically, you should check if the parent is non-null.

**Review by domenic — APPROVED** [review]
LGTM with nits; thanks for working on this!

**Comment by sideshowbarker** [comment]
> LGTM with nits; thanks for working on this!

Thanks — see the update I just now pushed, and lemme know if I got anything wrong

**Review by domenic — APPROVED** [review]

**Pull request merged**